cognee/modules/notebooks/methods/create_notebook.py

```
@@ -7,6 +7,38 @@
+            deletable=False,
+            force=force_refresh,
+        )
+
+        # Add to session and commit
+        session.add(notebook)
+        await session.commit()
+
+    except Exception as e:
+        print(f"Failed to fetch tutorial notebook from {TUTORIAL_ZIP_URL}: {e}")
+
+        raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)